bundles/org.openhab.binding.elgatokeylight/src/main/java/org/openhab/binding/elgatokeylight/internal/ElgatoKeyLightHandler.java

```diff
@@ -0,0 +1,181 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.elgatokeylight.internal;
+
+import static org.openhab.binding.elgatokeylight.internal.ElgatoKeyLightBindingConstants.CHANNEL_BRIGHTNESS;
+import static org.openhab.binding.elgatokeylight.internal.ElgatoKeyLightBindingConstants.CHANNEL_COLOR_TEMPREATURE;
+import static org.openhab.binding.elgatokeylight.internal.ElgatoKeyLightBindingConstants.CHANNEL_SWITCH;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.IncreaseDecreaseType;
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.PercentType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.elgatokeylight.internal.ElgatoKeyLight.LightStatus;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link ElgatoKeyLightHandler} is responsible for handling commands, which
+ * are sent to one of the channels.
+ *
+ * @author Gunnar Wagenknecht - Initial contribution
+ */
+@NonNullByDefault
+public class ElgatoKeyLightHandler extends BaseThingHandler {
+
+    private static final Logger LOG = LoggerFactory.getLogger(ElgatoKeyLightHandler.class);
```

Comment:
Loggers should not be static. See https://www.openhab.org/docs/developer/guidelines.html#f-logging